onnxruntime/core/framework/debug_node_inputs_outputs_utils.h

```diff
@@ -34,10 +38,14 @@ constexpr const char* kNameFilter = "ORT_DEBUG_NODE_IO_NAME_FILTER";
 constexpr const char* kOpTypeFilter = "ORT_DEBUG_NODE_IO_OP_TYPE_FILTER";
 // set to non-zero to dump data to files instead of stdout
 constexpr const char* kDumpDataToFiles = "ORT_DEBUG_NODE_IO_DUMP_DATA_TO_FILES";
+// set to non-zero to dump data to sqlite3 db instead of files
```

Comment:
>instead of files [](http://example.com/codeflow?start=46&length=16)

what about having a single environment variable to specify the destination? e.g., ORT_DEBUG_NODE_IO_DUMP_DATA_TO_DESTINATION with multiple valid values like "file", "sqlite", "stdout"
might be easier to use than multiple boolean variables